common/middleware/breadcrumbs.js

```diff
@@ -81,12 +81,12 @@ function setHome(options = {}) {
   const homeHref = options.href || '/'
 
   return function (req, res, next) {
-    const homeBreadcrumb = find(res.breadcrumb(), breadcrumb => {
+    const homeBreadcrumb = find(req.breadcrumb(), breadcrumb => {
       return breadcrumb._home
     })
 
     if (!homeBreadcrumb) {
-      res.breadcrumb({
+      req.breadcrumb({
```

Comment:
I think if we're deciding we do want this approach and to give up being able to chain `breadcrumb` and `render` together then we could in fact switch to just using this [express-breadcrumbs](https://www.npmjs.com/package/express-breadcrumbs) package.

I now have a vague memory of us writing this version for the uktrade app so that we can could `res.breadcrumb().render()`.

Without it, there's probably no point supporting that code ourselves.

Reply:
@solidgoldpig 's original point was that the pattern we're using is put things on `req` and then fill `res.locals` just before rendering.

In the `Basm` app do we have any other examples of this `res` chaining?

Reply:
re: [express-breadcrumbs](https://www.npmjs.com/package/express-breadcrumbs) - I really dislike reinventing the wheel if someone else already has, however that code hasn't been updated in 7 years, and talks about supporting Jade (not even Pug!), and has no tests, so I'm a little wary of adopting it!

Reply:
Yes, I think the lack of tests may have been why we wrote it ourselves in the UK Trade app.

>In the Basm app do we have any other examples of this res chaining?

I see this as a different use-case than the other instances though. The other instances are objects that we set and use later in the request lifecycle. "Request" being the key point there. 

The breadcrumbs I felt were more attached to the response, much like `res.render()` render a template. Probably a similar reason why they didn't go with `req.render()` in express.

I'll leave the call up to you on this one though.

Reply:
I've removed the commit that switched from `res` to `req`